### DISCUSSION TITLE
DM Access for $100+ Sponsors?

### DISCUSSION THREAD

**Discussion by scottmas** [Acly/krita-ai-diffusion#1428] [General]
Hey Acly! Just sponsored you at $100/month and was wondering if you would by chance be willing to let sponsors at that tier or above DM you on Discord or anything for occasional questions! My email is scottmortonashton@gmail.com, Discord handle @scottmas

Thanks for all you do. Your tools in Krita enable THE best professional workflows in AI art right now.

**Comment by Acly** [comment]
Sounds good. The project's Discord is https://discord.gg/pWyzHfHHhU - you can find me there and start a DM.
Thank you for the support!

**Comment by scottmas** [comment]
Oh I feel dumb. For the life of me I could not find ANY way to contact you and I thought you were maybe just being mysterious 😂. Thank you!
